massa-bootstrap/src/client_binder.rs

```diff
@@ -118,7 +145,11 @@ impl<D: Duplex> BootstrapClientBinder<D> {
 
     #[allow(dead_code)]
```

Comment:
Can be removed

Reply:
https://github.com/massalabs/massa/pull/3745/commits/5a990a2ad6be4038c2c9b6a85e0fc04c404f280e